web/themes/custom/subtheme_videogame/src/components/field/_mediaquery.scss

```diff
@@ -0,0 +1,46 @@
+@media screen and (max-width: 576px) {
+  .user {
+    &__field {
+      &-first-name,
+      &-last-name,
+      &-self-summary {
+        flex-direction: column;
+        text-align: center;
+
+        .field__label {
+          margin-bottom: 0;
+        }
+      }
+
+      &-first-name,
+      &-last-name,
+      &-self-summary {
+        .field__item {
+          margin: 0 1rem 2rem 1rem;
+        }
+      }
+    }
+  }
+
+  header .orangeupper,
+  footer .btn-secondary {
+    display: flex;
+    justify-content: center;
+  }
+
+  .views-view-responsive-grid .views-col {
+    width: 100% !important;
```

Comment:
What is you remove `.views-col` class from the grid? Then you wouldn't need to override rules.

Reply:
If I remove .views-col, the pictures won't float left, since it is defined in the Responsive Views Grid's scss file like the following:
.views-view-responsive-grid .views-col {
  float: left;
}